theano/sandbox/cuda/tests/test_dnn.py

```diff
@@ -823,7 +823,7 @@ def test_batchnorm_inference():
                 utt.assert_allclose(outputs[3], outputs[3 + 5])  # dscale
                 utt.assert_allclose(outputs[4], outputs[4 + 5])  # dbias
                 utt.assert_allclose(outputs[5], outputs[5 + 5])  # dmean
-                utt.assert_allclose(outputs[6], outputs[6 + 5])  # dvar
+                utt.assert_allclose(outputs[6], outputs[6 + 5], rtol=2e-3, atol=5e-5)  # dvar
```

Comment:
Yes, the daily buildbot have changed computer and GPUs (now GTX750). But
travis don't have GPUs.

We started to implement jenkins, it work for whitelisted people but the GPU
isn't enabled yet on job. But the computer also have 750.

I agree we should check if the rtol is at the minimim needed or not.

On Fri, Jul 22, 2016 at 6:17 AM, Jan Schlüter <notifications@github.com>
wrote:

> In theano/sandbox/cuda/tests/test_dnn.py
> <https://github.com/Theano/Theano/pull/4772#discussion_r71857582>:
>
> > @@ -823,7 +823,7 @@ def test_batchnorm_inference():
> >                  utt.assert_allclose(outputs[3], outputs[3 + 5])  # dscale
> >                  utt.assert_allclose(outputs[4], outputs[4 + 5])  # dbias
> >                  utt.assert_allclose(outputs[5], outputs[5 + 5])  # dmean
> > -                utt.assert_allclose(outputs[6], outputs[6 + 5])  # dvar
> > +                utt.assert_allclose(outputs[6], outputs[6 + 5], rtol=2e-3, atol=5e-5)  # dvar
>
> Does the buildbot run cuDNN code now? The rtol seems pretty high to me,
> is this the minimum that works? (Just making sure. The failure message
> should tell the relative and absolute difference.)
>
> —
> You are receiving this because you commented.
> Reply to this email directly, view it on GitHub
> <https://github.com/Theano/Theano/pull/4772/files/763b4b78d0ba1f813e8bd97a8a31a9d55f0759a7#r71857582>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AALC-wYvBEEJB7FpdKecZhLvV20R1--iks5qYJihgaJpZM4JSKp8>
> .
>